src/site/_collections/authors.js

```diff
@@ -117,7 +118,16 @@ module.exports = (collections) => {
       invalidAuthors.push(key);
     }
 
-    const authorsImage = path.join('/images', 'authors', `${key}@2x.jpg`);
+    let authorsImage = path.join('/images', 'authors', `${key}@2x.jpg`);
+    if (process.env.ELEVENTY_ENV === 'prod') {
+      const authorsImageExists = fs.existsSync(path.join('src', authorsImage));
+      if (!authorsImageExists) {
+        console.warn(
+          `No 2x image was found for ${author.title} (${author.key}), replacing with placeholder.`,
+        );
+        authorsImage = path.join('/images', 'authors', 'no-photo@2x.jpg');
```

Comment:
total nit: can we use the SVG here? It's smaller and nicer when the image is actually geometric.

Reply:
Yes we can! And I just updated it to do that.